subsys/disk/disk_access_stm32_sdmmc.c

```diff
@@ -64,6 +64,32 @@ static int stm32_sdmmc_clock_enable(struct stm32_sdmmc_priv *priv)
 		;
 
 	LL_RCC_SetSDMMCClockSource(LL_RCC_SDMMC1_CLKSOURCE_PLLSAI1);
+#elseif CONFIG_SOC_SERIES_STM32H7X
+	/* Configure PLL2 */
+	/* Select clock source */
+	/* Init pre divider DIVM */
+	LL_RCC_PLL2_SetM(CONFIG_CLOCK_STM32_PLL2_M_DIVISOR);
+	
+	/* Config PLL */
+	/* VCO sel, VCO range */
+	LL_RCC_PLL2_SetVCOInputRange(LL_RCC_PLLINPUTRANGE_1_2);
+	LL_RCC_PLL2_SetVCOOutputRange(LL_RCC_PLLVCORANGE_WIDE);
+
+	/* FRACN disable DIVP,DIVQ,DIVR enable*/
+	LL_RCC_PLL2FRACN_Disable();
+	LL_RCC_PLL2P_Enable();
+	LL_RCC_PLL2Q_Enable();
+	LL_RCC_PLL2R_Enable();
+
+	/* DIVN,DIVP,DIVQ,DIVR div*/
+	LL_RCC_PLL2_SetN(CONFIG_CLOCK_STM32_PLL2_N_MULTIPLIER);
+	LL_RCC_PLL2_SetP(CONFIG_CLOCK_STM32_PLL2_P_DIVISOR);
+	LL_RCC_PLL2_SetQ(CONFIG_CLOCK_STM32_PLL2_Q_DIVISOR);
+	LL_RCC_PLL2_SetR(CONFIG_CLOCK_STM32_PLL2_R_DIVISOR);
+	
+	LL_RCC_PLL2_Enable();
+	
+	LL_RCC_SetSDMMCClockSource(LL_RCC_SDMMC_CLKSOURCE_PLL2R);
```

Comment:
Since PLL configuration is done at another place, check that right configuration is enabled (CLOCK_STM32_PLL2_ENABLE)